media-gfx/imagemagick/imagemagick7-7.0.7.8.recipe

```diff
@@ -8,11 +8,11 @@ draw text, lines, polygons, ellipses and Bézier curves."
 HOMEPAGE="https://www.imagemagick.org/"
 COPYRIGHT="1999-2017 ImageMagick Studio LLC"
 LICENSE="IMAGEMAGICK"
-REVISION="1"
+REVISION="2"
 imageMagickVersion="${portVersion%.*}-${portVersion##*.}"
 SOURCE_URI="https://www.imagemagick.org/download/releases/ImageMagick-$imageMagickVersion.tar.xz
 	https://mirror.sobukus.de/files/src/imagemagick/ImageMagick-$imageMagickVersion.tar.xz"
-CHECKSUM_SHA256="77e9798ae32140d36a3a7cd09cb1fb6403c3f4482db8cc4a66ade69ebb5a5240"
+CHECKSUM_SHA256="547826f95a23ad3a1a6a514c0bff16ae875f6b7211809dff566e5994f65a0313"
```

Comment:
The file is no longuer available. You should probably rename the recipe to `imagemagick7-7.0.7.16.recipe`, reset REVISION to 1 and use:
```
CHECKSUM_SHA256="9b3ef8c8a8d2eafc0b72ea9f037720e5d50d5c072cc2847e7d97f65fd3af1f14"
```